hexrd/ui/image_tab_widget.py

```diff
@@ -165,9 +171,11 @@ def show_cartesian(self):
         self.image_canvases[0].show_cartesian()
         self.addTab(self.image_canvases[0], '')
         self.tabBar().hide()
+        self.switch_toolbar(self.currentIndex())
```

Comment:
By the way, in case we want to look at the canvas clearing sometime in the future, it looks like it is caused by this `self.switch_toolbar()` function.